content/applications/general/voip/axivox/vm_audio_messages.rst

```diff
@@ -0,0 +1,216 @@
+
+Voicemail
+=========
+
+The next step is to set up the individual voicemail boxes on the Axivox administrative portal. To
+access the portal visit `manage.axivox.com <axivox_admin_>`_ and login. Navigate to
+:guilabel:`Voicemails` in the left menu. If the voicemail option was activated in the user profile
+using this process :ref:`axivox/activate_voicemail`, then a voicemail is automatically created in
+this :menuselection:`Voicemails` section.
+
+.. tip::
+   It should be noted that some of the administrative portal language will be in French as Axivox is
+   a Belgian company. The global language is still set to one of the four options as seen here:
+   :ref:`axivox/global_language`
+
+Manually create voicemail
+-------------------------
+
+To manually create a new voicemail box, click on :guilabel:`Add a voicemail` or to edit an existing
+voicemail box click :guilabel:`Edit` to the far right of the existing voicemail box.
+
+.. example::
+   Suppose a sales or support team needs a general voicemail box. The voicemail would need to be
+   created manually and attached to an incoming number.
+
+The newly manually created voicemail box should be attached to an incoming number so that it can
+receive messages. To do so, first, navigate to :guilabel:`Incoming numbers` in the left menu. Then
+click :guilabel:`Edit` to the far right of the specific number that the voicemail should be linked
+to.
+
+Under :guilabel:`Destination type for voice call` click the drop-down menu and select
+:guilabel:`Voicemail`. Then select the drop-down menu on the next line labeled
+:guilabel:`Voicemail`, and select the manually created voicemail box. Enter a :guilabel:`Destination
+email address for Incoming SMS` and click :guilabel:`Save`. :guilabel:`Apply changes` in the upper
+right side of the screen to implement the change into production.
+
+.. tip::
+   Under the field labeled: :guilabel:`Destination email address for Incoming SMS` enter an email
+   to which incoming text messages sent to the incoming number can be received. Some incoming
+   numbers (US +1) in Axivox are capable of receiving text messages from individuals and automated
+   numbers. Should this field be left empty then the default destination address is used instead
+   (as previously set here in the start of manually creating a voicemail). To determine whether an
+   incoming number is capable of receiving SMS/text messages click on :guilabel:`Incoming numbers`
+   in the left menu. Then check the :guilabel:`SMS compatible` column for the incoming number.
+
+Notifications
+-------------
+
+Anytime a voicemail is received now on any of the automatically pre-configured or manually linked
+voicemail boxes, an email is sent to the user's email address as listed in the
+:menuselection:`Voicemails` screen or in the user's Axivox profile. This can be accessed by
+navigating to :guilabel:`Users` in the left menu and clicking on :guilabel:`Edit` next to the
+specific user in question.
```

Comment:
```suggestion
The next step is to set up the individual voicemail boxes on the Axivox administrative portal. To
access the portal, visit `manage.axivox.com <axivox_admin_>`_ and log in. Then, navigate to
:menuselection:`Voicemails`, located in the menu on the left.

If the voicemail option was activated in the user profile, using this process
:ref:`axivox/activate_voicemail`, then a voicemail is automatically created on the
:guilabel:`Voicemails` pages.

.. tip::
   It should be noted that some of the administrative portal language is in French, as Axivox is a
   Belgian company. The global language is still set to one of the four options as seen here:
   :ref:`axivox/global_language`

Manually create voicemail
-------------------------

To manually create a new voicemail box, click :guilabel:`Add a voicemail` on the
:guilabel:`Voicemails` page. Or, edit an existing voicemail box, by clicking :guilabel:`Edit` to the
far-right of an existing voicemail box on the :guilabel:`Voicemails` page.

.. example::
   Suppose a sales or support team needs a general voicemail box. The voicemail would need to be
   created manually, and attached to an incoming number.

The new, manually-created voicemail box should be attached to an incoming number, so it can receive
messages. To do so, navigate to :menuselection:`Incoming numbers`, located in the menu on the left.
Then, click :guilabel:`Edit` to the far-right of the specific number the voicemail should be linked
to.

In the :guilabel:`Destination type for voice call` field, click the drop-down menu, and select
:guilabel:`Voicemail`. Then, open the drop-down menu on the next line labeled,
:guilabel:`Voicemail`, and select the manually-created voicemail box.

.. important::
   If an incoming number is capabable of receiving SMS/text messages, an additional field,
   :guilabel:`Destination email address for Incoming SMS` is present.

   To determine whether an incoming number is capable of receiving SMS/text messages, click
   :guilabel:`Incoming numbers` from the menu on the left, then check the :guilabel:`SMS compatible`
   column for the incoming number.

Then, if applicable, in the field labeled, :guilabel:`Destination email address for Incoming SMS`,
enter an email to which incoming text messages sent to the incoming number can be received. Some
incoming numbers (US +1) in Axivox are capable of receiving text messages from individuals and
automated numbers.
   
Should this field be left empty, the default destination address is used, instead (as previously set
in the beginning of the process for manually creating a voicemail).

Once all desired configurations are complete, click :guilabel:`Save`, then click :guilabel:`Apply
changes` in the upper-right corner of the screen to implement the change into production.

Notifications
-------------

Now, whenever a voicemail is received on any of the automatically pre-configured, or
manually-linked, voicemail boxes, an email is sent to the user's email address, as listed in the
:guilabel:`Voicemails` page, or in the user's Axivox profile.

This information can be accessed by navigating to :menuselection:`Users` in the left menu, and
clicking :guilabel:`Edit` next to the specific user in question.
```